### PR TITLE
Alert counters

### PR DESCRIPTION
**Pull request by anmolbabu**

### CODE REVIEW

**Comment by anmolbabu** [comment]
Needs to be verified

File: tendrl/alerting/sync/__init__.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    ),
+                    {
+                        "message": 'Failed to update cluster alert counter.'
+                        ' Exception %s' % str(ex)
+                    }
+                )
+                continue
+
+    def _run(self):
+        while not self.complete.is_set():
```
Comment (r0h4n) [inline tendrl/alerting/sync/__init__.py]: umm, youll need to define a stop() which sets self.complete

File: tendrl/alerting/manager/__init__.py [hunk 2]
```diff
@@ -58,6 +61,7 @@ def start(self):
     def stop(self):
         try:
             self.watch_manager.stop()
+            self.sync_thread.stop()
```
Comment (r0h4n) [inline tendrl/alerting/manager/__init__.py]: there is no stop method for TendrlAlertingSync

File: tendrl/alerting/utils/central_store_util.py [hunk 3]
```diff
@@ -44,3 +70,69 @@ def get_alerts():
...
+
+
+def update_alert_counters(
+    entity_type,
+    crit_alert_count,
+    warn_alert_count,
+    **kwargs
+):
+    alert_counter_path = get_alert_counter_update_path(entity_type, **kwargs)
+    NS._int.client.write(
```
Comment (r0h4n) [inline tendrl/alerting/utils/central_store_util.py]: Please make objects out of alert counters, we need to stop doing direct write to etcd in new PRs, the object.save() handles a lot of etcd exceptions which arent handled here

File: tendrl/alerting/sync/__init__.py (hunk 1 above)
Comment (anmolbabu) [inline tendrl/alerting/sync/__init__.py]: Aah missed it !! Adding now...

File: tendrl/alerting/manager/__init__.py (hunk 2 above)
Comment (anmolbabu) [inline tendrl/alerting/manager/__init__.py]: Adding now...

File: tendrl/alerting/utils/central_store_util.py (hunk 3 above)
Comment (anmolbabu) [inline tendrl/alerting/utils/central_store_util.py]: Done

**Comment by anmolbabu** [comment]
@r0h4n @nthomas-redhat @shtripat Please review..

Verified...

**Comment by anmolbabu** [comment]
@anivargi This PR adds alert counters in etcd for 
1. cluster level:
    /clusters/{cluster_id}/alert_counters/critical_count for critical alerts count
   /clusters/{cluster_id}/alert_counters/warning_count for warning_count
2. node level:
    /nodes/{node_id}/alert_counters/critical_count for critical alerts count
   /nodes/{node_id}/alert_counters/warning_count for warning_count
I would assume this will be automatically made available via GetClusterList Api please confirm the same...
If not readily available through GetClusterList API, please do the needful to make it available so that the UI can make use of this in the cluster listing api. cc: @gnehapk @cloudbehl

**Pull request merged**